### DISCUSSION TITLE
Adding left hand tweak

### DISCUSSION THREAD

**Discussion by b34n5pr072** [BedrockTweaks/Files#253] [Ideas]
Adding a left hand tweak like java in character settings would be great.
There is already one, but it was 2 years ago and the angle isn't right, and not even half the items are done, and it doesn't support custom items. maybe the way it could be coded is just move the fists.

**Comment by SquatchHunter** [comment]
not possible
everything breaks

anything we add left hand function to will break, we cannot over write vanilla items and would be required to _remake_ every vanilla item, which would then be technically different to the current vanilla items and so they wouldnt stack ect ect ect

**Comment by b34n5pr072** [comment]
shucks

**Comment by SquatchHunter** [comment]
good news, the new data driven items coming down the pipe line _may_ make this possible, but we will see
